### PR TITLE
Linking my blog to the website

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: Screenshot(21).jpg
```
Comment (nikit19) [inline _data/blogs.yml:228]: Please upload this image

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make this change

File: _data/blogs.yml (hunk 1 above)
Comment (hshafaq7) [reply _data/blogs.yml:228]: I am not able to upload the image.Please Help!!

Comment (hshafaq7) [reply _data/blogs.yml:228]: Please tell me how do I do it.

Comment (codedsun) [reply _data/blogs.yml:228]: @hshafaq7 Kindly add the image to the following folder and update the commit 👍 tell us if you are facing any issue.

Comment (hshafaq7) [reply _data/blogs.yml:228]: Which Folder

**Comment by ParthS007** [comment]
closing as a duplicate of #762

**Pull request closed without merging**